src/components/DataIndicators/index.js

```diff
@@ -0,0 +1,49 @@
+import { Hidden } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Desktop from "./Desktop";
+import Mobile from "./Mobile";
+import Tablet from "./Tablet";
+import useStyles from "./useStyles";
+
+import Section from "@/pesayetu/components/Section";
+
+const DataIndicators = ({ title, items, ...props }) => {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Section>
```

Comment:
thiis component is not limited with our defined section widths
![Screenshot from 2021-08-11 16-23-12](https://user-images.githubusercontent.com/7962097/129037006-ebdfc969-7103-4340-8fd9-5bbd3027b8ae.png)

Reply:
Got it